npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -210,6 +211,18 @@ func (iMgr *IPSetManager) AddToSets(addToSets []*IPSetMetadata, ip, podKey strin
 	if len(addToSets) == 0 {
 		return nil
 	}
+
+	// possible formats
+	// 192.168.0.1
+	// 192.168.0.1,tcp:25227
+	// always guaranteed to have ip, not guaranteed to have port + protocol
+	ipDetails := strings.Split(ip, ",")
+	if len(ipDetails) > 0 && !util.IsIPV4(ipDetails[0]) {
```

Comment:
i think this should be in ipsetmanager, because this is relevant to sets validations. CIDR block checks should also be moved one step below into ipsetmanager if you ask me.

Reply:
we can skip the length check since split always returns a slice length 1+